draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1145,8 +1156,8 @@ This document does not cover serialization or deserialization.
 
 ### Domain separation {#term-domain-separation}
 
-Cryptographic protocols that use random oracles are often analyzed
-under the assumption that random oracles answer only queries generated
+Cryptographic protocols whose security relies on random oracles are often analyzed
```

Comment:
It this more accurate?

```suggestion
Cryptographic protocols proven secure in the random oracle model are often analyzed
```

Reply:
I pushed another commit that addresses this.